docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -63,3 +63,61 @@ For example:
 
 The same applies to `slotProps` and `componentsProps`.
 :::
+
+### Adapters
+
+:::success
+The following breaking changes only impact you if you are using the adapters outside the pickers like displayed in the following example:
+
+```tsx
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+
+const adapter = new AdapterDays();
+adapter.isValid(dayjs('2022-04-17T15:30'));
+```
+
+If you are just passing an adapter to `LocalizationProvider`, then you can safely skip this section.
+:::
+
+#### Restrict the input format of the `isValid` method
+
+The `isValid` method used to accept any type of value and tried to parse them before checking their validity.
+The method has been simplified and now only accepts an already-parsed date or `null` (ie: the same formats used by the `value` prop in our components)
```

Comment:
```suggestion
The method has been simplified and now only accepts an already-parsed date or `null`.
Which is the same type as the one accepted by the components `value` prop.
```


Avoid "our", and ie (Google recommend to avoid latin expressions)